README.md

```diff
@@ -109,6 +109,11 @@ module ThingsRepresenter
 end
 ```
 
+RELEASING
+---------
+
+Edit `Rakefile` and run `rake` (because we use gemhadar).
```

Comment:
Maybe you want to mention, what rake commands to run, usually one of `rake version:bump:*`, `rake build`, until you're happy with the placement of the tag, and then `rake push` to push the gem and the tags to remote.